src/components/library/EditLyrics.vue

```diff
@@ -62,7 +62,9 @@
             </div>
           </div>
 
-          <div class="h-full grow overflow-hidden">
+          
+          // NOTE: AsyncCodemirror component does not have @wheel event handler, so it has to be handled here (in the container)
```

Comment:
Comments on HTML should be in the format `<!-- COMMENT GOES HERE -->`, otherwise they would render to browser/UI.

```suggestion
          <!-- NOTE: AsyncCodemirror component does not have @wheel event handler, so it has to be handled here (in the container) -->
```